lib/PrintAsClang/PrintClangValueType.cpp

```diff
@@ -25,9 +24,22 @@
 using namespace swift;
 
 void ClangValueTypePrinter::printStructDecl(const StructDecl *SD) {
+  auto structABIInfo = interopContext.getStructDeclInfo(SD);
+  if (!structABIInfo) {
+    // FIXME: handle non-fixed layout structs.
```

Comment:
Does it make sense to print out something here like `<resilient-struct>`?

Reply:
Maybe print out an unavailable forward class declaration?